web/src/project/ProjectSummary.tsx

```diff
@@ -348,7 +348,11 @@ export const ProjectSummary = () => {
                             if (b[0] === "no-batch") {
                                 return -1;
                             }
-                            return a[0] > b[0] ? 1 : -1;
+                            if (isNaN(+a[0]) && isNaN(+b[0])) {
+                                //if both numbers
+                                return +a[0] - +b[0];
```

Comment:
I don't understand this, are you using the `+` operator to coerce strings to integers to check if they're numbers. Is there a cleaner way, if not could you clarify why you do it this way in a comment?

Reply:
Fixed :)